packages/docusaurus/bin/docusaurus.js

```diff
@@ -49,9 +61,11 @@ if (notifier.update && notifier.update.current !== notifier.update.latest) {
   const docusaurusUpdateMessage = boxen(
     `Update available ${chalk.dim(`${notifier.update.current}`)}${chalk.reset(
       ' → ',
-    )}${chalk.green(`${notifier.update.latest}`)}\nRun ${chalk.cyan(
-      'yarn upgrade @docusaurus/core',
-    )} to update`,
+    )}${chalk.green(
+      `${notifier.update.latest}`,
+    )}\n\nTo upgrade Docusaurus packages with the latest version, run the following command:\n${chalk.cyan(
+      `yarn upgrade ${siteDocusaurusPackagesForUpdate}`,
```

Comment:
Was wondering, is there an npm equivalent we could use here?

Reply:
Done, take a look please.